crates/ruff_server/src/server/api.rs

```diff
@@ -55,6 +55,7 @@ pub(super) fn request<'a>(req: server::Request) -> Task<'a> {
     }
     .unwrap_or_else(|err| {
         tracing::error!("Encountered error when routing request with ID {id}: {err}");
+        show_err_msg!("Ruff failed to handle a request from the editor. Check the error log for more details.");
```

Comment:
nit: I'd remove the "error" part

```suggestion
        show_err_msg!("Ruff failed to handle a request from the editor. Check the logs for more details.");
```